packages/fnr-perf/project.json

```diff
@@ -5,15 +5,11 @@
   "projectType": "library",
   "targets": {
     "build": {
-      "executor": "@nx/vite:build",
-      "outputs": ["{options.outputPath}"],
+      "executor": "nx:run-commands",
       "options": {
-        "outputPath": "dist/packages/fnr-perf",
-        "generatePackageJson": true,
-        "externalDependencies": "none",
-        "updateBuildableProjectDepsInPackageJson": true,
-        "buildableProjectDepsInPackageJsonType": "dependencies"
-      }
+        "command": "tsc -p packages/fnr-perf/tsconfig.lib.json",
+      },
+      "dependsOn": ["build-browser"]
```

Comment:
The build-browser target should also be added, also take a look at the package.json in mui, I forgot some config from there is actually needed, specifically you will need to add this: https://github.com/unfoldingWord/findr/blob/131df9c8f6e8e4cfe3313d40ad01e9cec56ad9d1/packages/fnr-mui/package.json#L4C3-L6C27 to findr-perf package.json

Reply:
Will do! Thanks! Could you explain what this is doing? `build-browser` seems ambiguous as I'm sure you're not intending to build an entire browser!

Reply:
it could be renamed to build-for-browser, it is just using a different configuration that outputs es modules in contrast to the default nodejs config which outputs commonjs.